client/javascripts/sagas/auth.js

```diff
@@ -0,0 +1,19 @@
+import { call, takeEvery } from 'redux-saga/effects';
+import * as api from 'api';
+import {
```

Comment:
__[eslint]__
* Error - 'actions' should be listed in the project's dependencies. Run 'npm i -S actions' to add it (import/no-extraneous-dependencies)

<!-- Comment by [SideCI](https://sideci.com) -->